package exceptionTask;

public class exceptionTwo {
	
	public static void main(String[] args) {
		
		int a, b, c = 0;
	}

}
